packages/admin-ui/ui/public/locales/fr/translation.json

```diff
@@ -0,0 +1,1919 @@
+  "help-dialog-subject": "Sujet",
+  "help-dialog-what-is-it-about": "De quoi s'agit-il ?...",
+  "help-dialog-write-a-message": "Écrire un message...",
+  "help-dialog-feel-free-to-join-our-community-of": "N'hésitez pas à rejoindre notre communauté de",
+  "help-dialog-merchants-and-e-commerce-developers": "commerçants et développeurs de commerce électronique",
+  "help-dialog-send-a-message": "Envoyer un message",
+  "invite-modal-success": "Succès",
+  "invite-modal-invitation-sent-to": "Invitation envoyée à {{user}}",
+  "invite-modal-error": "Erreur",
+  "invite-modal-member": "Membre",
+  "invite-modal-admin": "Administrateur",
+  "invite-modal-developer": "Développeur",
+  "invite-modal-invite-users": "Inviter des utilisateurs",
+  "invite-modal-email": "E-mail",
+  "invite-modal-role": "Rôle",
+  "invite-modal-select-role": "Sélectionner le rôle",
+  "invite-modal-cancel": "Annuler",
+  "invite-modal-invite": "Inviter",
+  "login-card-no-match": "Ces identifiants ne correspondent pas à nos enregistrements.",
+  "login-card-log-in-to-medusa": "Connectez-vous à Medusa",
+  "login-card-email": "E-mail",
+  "login-card-password": "Mot de passe",
+  "login-card-forgot-your-password": "Mot de passe oublié ?",
+  "metadata-add-metadata": "Ajouter des métadonnées",
+  "product-attributes-section-edit-attributes": "Modifier les attributs",
+  "product-attributes-section-dimensions": "Dimensions",
+  "product-attributes-section-configure-to-calculate-the-most-accurate-shipping-rates": "Configurer pour calculer les tarifs d'expédition les plus précis",
+  "product-attributes-section-customs": "Douanes",
+  "product-attributes-section-cancel": "Annuler",
+  "product-attributes-section-save": "Enregistrer",
+  "product-attributes-section-mid-code": "Code MID",
+  "product-attributes-section-hs-code": "Code HS",
+  "product-attributes-section-country-of-origin": "Pays d'origine",
+  "product-general-section-success": "Succès",
+  "product-general-section-successfully-updated-sales-channels": "Mise à jour des canaux de vente réussie",
+  "product-general-section-error": "Erreur",
+  "product-general-section-failed-to-update-sales-channels": "Échec de la mise à jour des canaux de vente",
+  "product-general-section-edit-general-information": "Modifier les informations générales",
+  "product-general-section-gift-card": "Carte-cadeau",
+  "product-general-section-product": "Produit",
+  "product-general-section-metadata": "Métadonnées",
+  "product-general-section-cancel": "Annuler",
+  "product-general-section-save": "Enregistrer",
+  "product-general-section-delete": "Supprimer",
+  "product-general-section-edit-sales-channels": "Modifier les canaux de vente",
+  "product-general-section-published": "Publié",
+  "product-general-section-draft": "Brouillon",
+  "product-general-section-details": "Détails",
+  "product-general-section-subtitle": "Sous-titre",
+  "product-general-section-handle": "Poignée",
```

Comment:
todo: this is a direct translation it does not work. "Identifiant unique" peut etre? @olivermrbl